### ISSUE TITLE
'mediacapture-streams/MediaStream-idl.https.html' requires event that was removed from the spec

### ISSUE THREAD

**Issue by eric-carlson** [web-platform-tests/wpt#15560]
mediacapture-streams/MediaStream-idl.https.html ends when the 'inactive' event is fired:

```
      stream.oninactive = t.step_func(() => {
        assert_false(stream.active);
        stream.getTracks().forEach(track => {
          assert_equals(track.readyState, "ended");
        });
        // Use a timeout to detect a misfire of the ended event.
        t.step_timeout(t.step_func_done());
      });

```

but the 'active' and 'inactive' were [removed from the spec](https://github.com/w3c/mediacapture-main/pull/291) to fix issue [#283](https://github.com/w3c/mediacapture-main/issues/283).

**Comment by guest271314** [comment]
Note, `onactive` and `oninactive` are still properties of `MediaStream` instance at Chromium 72.

**Comment by guest271314** [comment]
@eric-carlson The results of the various `MediaStreamTrack` and `MediaStream` events vary widely at the front-end between Chromium (76) and Firefox (69). 

For example, at Chromium 76 `mute` event is dispatched at `MediaStreamTrack` of `canvas.captureStream()` when images are not longer set at source `<canvas>` element; Firefox (Nightly) does not dispatch `mute` event. 

Specification https://w3c.github.io/mediacapture-main/#event-mediastreamtrack-mute 

> Fired when... object's source is temporarily unable to provide data. 



At Chromium `inactive` event is still defined at `MediaStream` and is dispatched when the single `MediaStreamTrack` is removed from the `MediaStream`. Firefox _does not_ have `inactive` event defined. 

In some cases the `ended` event of a `MediaStreamTrack` of `kind` `"video"` _is not_ dispatched at either Chromium or Firefox where no further data is being provided, `MediaStream.getTrack()` returns `[]`, `MediaStreamTrack.enabled` is `false`, `MediaStreamTrack.readyState` is `"ended"`, and `MediaStream.active` is `false` (related https://github.com/w3c/mediacapture-fromelement/issues/77; https://github.com/w3c/mediacapture-fromelement/issues/78; https://github.com/w3c/mediacapture-fromelement/issues/80).

https://w3c.github.io/mediacapture-main/#event-summary

> The MediaStreamTrack object's source will no longer provide any data, either because the user revoked the permissions, or because the source device has been ejected, or because the remote peer permanently stopped sending data.





The current Editor's Draft of the specification https://w3c.github.io/mediacapture-main

> A MediaStream object is said to be active when it has at least one MediaStreamTrack that has not ended. A MediaStream that does not have any tracks or only has tracks that are ended is inactive.

**Comment by dontcallmedom** [comment]
this got fixed in #17491
